Source/FakeItEasy/Core/DefaultReturnValueRule.cs

```diff
@@ -17,6 +17,17 @@ private static IFakeAndDummyManager FakeManager
             get { return ServiceLocator.Current.Resolve<IFakeAndDummyManager>(); }
         }
 
+        public static object ResolveReturnValue(IInterceptedFakeObjectCall fakeObjectCall)
+        {
+            object result;
+            if (!FakeManager.TryCreateDummy(fakeObjectCall.Method.ReturnType, out result))
+            {
+                return fakeObjectCall.Method.ReturnType.GetDefaultValue();
+            }
+
+            return result;
+        }
+

```

Comment:
I considered whether to move `ResolveReturnValue` to another location, but it felt like it _wanted_ to belong to `DefaultReturnValueRule`.
Happy to put somewhere else if the "one rule calling another's static method" doesn't sit well.